util/util.go

```diff
@@ -195,7 +196,8 @@ func (ns *NodeSpec) Load() (IpfsNode, error) {
 	switch ns.Type {
 	case "local":
 		ln := &LocalNode{
-			Dir: ns.Dir,
+			Dir:     ns.Dir,
+			Profile: ns.Extra["profile"].(string),
```

Comment:
Relying on this being set makes me a bit worried. Will this always be the case? We may want to default to "".